draft-ietf-dnsop-alt-tld.xml

```diff
@@ -79,7 +79,11 @@
       top-level label can be used as the final (rightmost) label to signify
       that the name is not rooted in the global DNS, and that it should not be
       resolved using the DNS protocol.</t>
-
+      
+      <t>In <xref target="iana-6761"/>, the IANA is requested to add the .alt name
+      to the "Special-Use Domain Name" registry. Such addition will prevent the
+      name from appearing in the global DNS root zone.</t>
```

Comment:
"Such addition will prevent the name from appearing in the global DNS root zone." - citation needed :-)

I fully agree that from a practical point this is true, but I don't think that there is anything that we can point at that states this clearly. E.g. the last Applicant Guidebook (Sec 2.2.1.2.1 Reserved Names) copied and pasted name from the registry. 
I don't really know to to address this, other than just dropping the sentence or perhaps something more hand-wavey like: "In <xref target="iana-6761"/>, the IANA is requested to add the .alt name to the "Special-Use Domain Name" registry. The expectation is that this name will never be delegated in the global DNS root zone.".... but I suspect that just dropping it is still much better.

Reply:
See revised wording in 1edfc7a